syntax = "proto3";

option go_package = "github.com/Lucino772/envelop/pkg/protobufs";

import "google/protobuf/empty.proto";

service Process {
    rpc WriteCommand (Command) returns (google.protobuf.Empty);
    rpc StreamLogs (google.protobuf.Empty) returns (stream Log);
}

message Command {
    string value = 1;
}

message Log {
    string value = 1;
}
